### PR TITLE
Repolished react example

### PR DESCRIPTION
**Pull request by alexalikiotis**

### CODE REVIEW

**Comment by alexalikiotis** [comment]
@helio-frota yeah I thought about that too. I thought to change the 'yarn' word to 'npm' but at the top says created using create-react-app and that uses yarn by default so that would be confusing. But the idea of removing the client doc all together sounds good to me 👍

File: react/client/package.json [hunk 1]
```diff
@@ -1,27 +1,36 @@
 {
-  "name": "react-opossum",
+  "name": "react",
```
Comment (lholmquist) [inline react/client/package.json:2]: I think the naming here is confusing snyk,  maybe it should be opossum-react-client.  Up to you if you want to change the server package.json to be something similar,  but not needed

Comment (alexalikiotis) [reply react/client/package.json]: I renamed it from **opossum-react** to **react-opossum** 😎

Comment (lholmquist) [reply react/client/package.json]: i'm getting some weird npm error during the npm install:

```
npm ERR! Cannot read property 'match' of undefined
```

i removed my local node_modules folder and the package-lock.json and it worked.

Comment (alexalikiotis) [reply react/client/package.json]: This error appeared with the last commit ?? (changing the name is package.json)

Comment (lholmquist) [reply react/client/package.json]: yea,  I would just delete your node_modules folder and package-lock.json  and then do another install to recreate the package-lock.json,  at least that is what the internet said :sunglasses:

Comment (alexalikiotis) [reply react/client/package.json]: yeah it seems to work now 👍

**Pull request merged**